src/actions/CryptoExchangeActions.js

```diff
@@ -29,6 +29,11 @@ export type SetNativeAmountInfo = {
   toPrimaryInfo?: GuiCurrencyInfo
 }
 
+const pluginToName = {
+  shapeshift: 'ShapeShift',
+  changelly: 'Changelly'
+}
```

Comment:
We already export this information on the account. `account.swapConfig[pluginName].swapInfo.displayName` will get it.